xml/hwbp_registers.xml

```diff
@@ -820,18 +871,27 @@
         tie any number of upper bits to 0. The $|select|$ bits may only support
         0 (ignore).
 
-        <field name="mvalue" bits="31:26" access="WARL" reset="0">
-            Data used together with \FcsrTextraThirtytwoMselect.
+        <field name="mhvalue" bits="31:26" access="WARL" reset="0">
+            Data used together with \FcsrTextraThirtytwoMhselect.
         </field>
-        <field name="mselect" bits="25" access="WARL" reset="0">
-            0: Ignore \FcsrTextraThirtytwoMvalue.
+        <field name="mhselect" bits="25:23" access="WARL" reset="0">
+            000: Ignore \FcsrTextraThirtytwoMhvalue.
 
-            1: This trigger will only match if the low bits of
-            \RcsrMcontext equal \FcsrTextraThirtytwoMvalue.
+            100: This trigger will only match if the low bits of
+            \RcsrMcontext/\RcsrHcontext equal \FcsrTextraThirtytwoMhvalue.
+
+            x01: This trigger will only match if the low bits of
+            \RcsrMcontext/\RcsrHcontext equal \{\FcsrTextraThirtytwoMhvalue, mhselect[2]\}.
```

Comment:
I assume this means 001 or 101. It's probably more clear to write "001/101" explicitly instead of relying on x01. Alternatively, you could explain what is meant by x01 in the Register Definition Format section.

Reply:
Even ignoring the wildcard, the rest of the document consistently uses decimal instead of binary.  So I changed it to list all the decimal values.  This means that the decimal values are not in order but the section is short enough that I don't think it's hard to comprehend and this keeps things reasonably compact.

I also fixed an incorrect reference to satp here.